core/test/functional/routes/api/posts_spec.js

```diff
@@ -7,6 +7,23 @@ var should = require('should'),
     ghost = testUtils.startGhost,
     request;
 
+function mobiledocDocument(content) {
```

Comment:
There are two things which are annoying in the test env.
Some of the tests are not using the data-generator directly (`e.g. DataGenerator.createPost`) to generate post content and the test env doesn't use our models to insert data (it uses `knex` directly). But we want to change this in the future. Right now, this eats too much time.

For now, i would put this function to `test/utils/fixtures/data-generator.js`, which looks like a useful place.

Then you can access the function with `testUtils.DataGenerator.transformIntoMobiledoc` or similar.

And in the future, we should always use the `DataGenerator.createPost` helper, pass only html/plaintext, which is automatically transformed into mobiledoc by our models layer.